odh/base/monitoring/overrides/prometheus-operator/base/prometheus-rules.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
```

Comment:
I know these labels maps to the `ruleSelector` in `kind: Prometheus`.

Is there any significance for the labels used? I'm not sure I understand "why" these labels and not "others". Why there are 2 different labels? That's just a question out of curiosity, not a critique. Maybe we can think of something more fitting to our usecase than these...

Reply:
So I just followed the recommended best practice based on the documentation [here](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/alerting.md#fire-alerts). According to which it mentions:

```
Use the label selector field ruleSelector in the Prometheus object to define the rule files that you want to be mounted into Prometheus.

The best practice is to label the PrometheusRules containing rule files with role: alert-rules as well as the name of the Prometheus object, prometheus: example in this case.
```

Reply:
What is the deployed Prometheus called?

Reply:
so [this](http://prometheus-portal-opf-monitoring.apps.cnv.massopen.cloud/) is our Prometheus, and we have defined the name as `odh-monitoring`: https://github.com/operate-first/apps/blob/master/odh/base/monitoring/overrides/prometheus-operator/base/prometheus.yaml#L7

Reply:
I think lets go with `prometheus: odh-monitoring`

Reply:
> retain it as prometheus: odh-monitoring

:+1:  to that. I like that

Reply:
updated :+1: